app/models/user.rb

```diff
@@ -1,14 +1,24 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :confirmable, :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable

```

Comment:
- [ ] As mentioned by the previous :point_up:  reviewer please ensure that the application asks the user for an email confirmation. Please include the Devise `:confirmable` option but still you didn't add  `:confirmable` so kindly add `:confirmable` :slightly_smiling_face: 

```suggestion
  devise :confirmable, :database_authenticatable, :registerable,
         :recoverable, :rememberable, :validatable, :confirmable

```

Reply:
@aamirkhan2478  Friend... did you see the code? what you ask me... it's there, you can confirm the email, and then it redirects you to the log in again!

Reply:
I made a mistake I didn't notice the `:confirmable` but it's there I am really sorry